rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -924,7 +922,7 @@ void Z3ConvVisitor::VisitUnaryApp(z3::expr z_op) {
         auto s_size{GetZ3SortSize(z_op)};
         auto t_op{ast_ctx->getIntTypeForBitwidth(s_size, /*sign=*/0)};
         c_op = ast.CreateCStyleCast(t_op, c_sub);
-      } else if (z_func_name == "BoolToBV" || z_func_name == "PtrDecay") {
+      } else if (z_func_name == "PtrDecay") {
```

Comment:
My recommendation would be make all of these strings into global `std::string`that represent your "abi" of sorts for interfacing with z3.

Reply:
Made into issue #136